exercises/dnd-character/README.md

```diff
@@ -0,0 +1,82 @@
+# Acronym
```

Comment:
Please regenerate the README using [`configlet`](https://github.com/exercism/configlet)

Reply:
I configured the configlet tool and have made the necessary changes. Does the configlet tool also edit the config.json or is that done manually? I cannot seem to find the options to change that correctly.

Reply:
config.json updated manually. Let me know if you agree with the topics section.